apps/modernization-ui/src/apps/page-builder/pages/ValuesetLibrary/ValuesetLibrary.tsx

```diff
@@ -60,7 +69,7 @@ export const ValuesetLibrary = ({ hideTabs, types, modalRef }: any) => {
     if (hideTabs) return <div className="valueset-local-library">{renderValueSetList}</div>;
 
     return (
-        <PageBuilder nav>
+        <PageBuilder nav={!hideTabs}>
```

Comment:
Issue: the `PageBuilder` component is somewhat deprecated and will be removed in the future. Would you mind removing its use from the `ValuesetLibrary` component?